replace_using_package_version/replace_using_package_version.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 #
-# Copyright (c) 2018 SUSE Linux GmbH.  All rights reserved.
+# Copyright (c) SUSE LLC.  All rights reserved.
```

Comment:
I think we should use everywhere

```
# Copyright (c) 2023 SUSE LLC
```

or

```
# Copyright (c) 2018 - 2023 SUSE LLC
```

Reply:
Copyright years are actually not really necessary, but let's keep them to make legal happy.